script/Arcueid/Day 7/07_00_ARC07_3_1_1.txt

```diff
@@ -29,8 +29,9 @@ I'll ask her about their relationship in more detail.
 -- Page 522, Offset 17867.
 -- 「お、さっすが人力だけでピラミッドを建てちゃう霊長類！　堅実かつ誠実ね。
 // What are we trying to get at with 堅実 vs/plus 誠実
+// dropped space before 'So' is intentional. Check in game if editing this line.
```

Comment:
Wait what? Why does this line render weird? This doesn't feel right

Reply:
it's just coincidental, but on reflection this is a bad solution without a manual newline. I'm just gonna revert because this is so fickle and minor.